docs/content/manual/manual.yml

```diff
@@ -3537,6 +3537,68 @@ sections:
               (.posts[] | select(.author == "stedolan") | .comments) |=
                   . + ["terrible."]
 
+  - title: Comments
+
+    body: |
+
+      You can write comments in your jq filters using `#`.
+
+      A `#` character (not part of a string) starts a comment.
+      All characters from `#` to the end of the line are ignored.
+
+      If the end of the line is preceded by an odd number of backslash
+      characters, the following line is also considered part of the
+      comment and is ignored.
+
+      For example, the following code outputs `[1,3,4,7]`
+
+          [
+            1,
+            # foo \
+            2,
+            # bar \\
+            3,
+            4, # baz \\\
+            5, \
+            6,
+            7
+            # comment \
+              comment \
+              comment
+          ]
+
+      Backslash continuing the comment on the next line can be useful
+      when writing the "shebang" for a jq script:
+
+          #!/bin/sh --
+          # sum - Output the sum of the given arguments (or stdin)
+          # usage: sum [numbers...]
+          # \
+          exec jq --args -MRnf "$0" -- "$@"
+
+          $ARGS.positional |
+          reduce (
+            if . == []
+              then inputs
+              else .[]
+            end |
+            . as $dot |
+            try
+              tonumber
+            catch
+              (@json "sum: Invalid number \($dot).\n" | halt_error(1))
+          ) as $n (0; . + $n)
```

Comment:
Hmm, I should probably make it reject nan or it will output null instead of a number. =)